src/module/actor/mixins/actor-damage.js

```diff
@@ -320,20 +320,21 @@ export const ActorDamageMixin = (superclass) => class extends superclass {
 
         if (!damage.isHealing) {
             /** Update temp hitpoints */
-            let newTempHP = Math.clamped(originalTempHP - remainingUndealtDamage, 0, actorData.attributes.hp.tempmax);
+            let newTempHP = Math.clamped(originalTempHP - remainingUndealtDamage, 0,
+                actorData.attributes.hp.tempmax ? actorData.attributes.hp.tempmax : actorData.attributes.hp.temp);
```

Comment:
```suggestion
let newTempHP = Math.clamped(originalTempHP - remainingUndealtDamage, 0, actorData.attributes.hp.tempmax || actorData.attributes.hp.temp);
```
This can be done a bit more concisely

Reply:
You are totally correct but I discovered it would be better to just remove the `+Max` field from the UI and not calculate with it anymore. Because it's not used in the starfinder system anyway. But since there are around 1400 lines in the code (mostly alien archive) having that property completely removing that thing from the code should be a different PR or maybe never done at all.

Reply:
> You are totally correct but I discovered it would be better to just remove the `+Max` field from the UI and not calculate with it anymore. Because it's not used in the starfinder system anyway. But since there are around 1400 lines in the code (mostly alien archive) having that property completely removing that thing from the code should be a different PR or maybe never done at all.

nvm found something that heals to a specific max amount of temp hp: https://aonsrd.com/ArmorUpgrades.aspx?ItemName=Brown&Family=Force%20Field
There might be more but this one is evidence enough.
Since there is currently nothing automated that heals tempHP to a maximum it's just not needed.
But if that ever gets automated the `hp.maxtemp` should be set inside the item/feat/object whatever that is providing heal to a specific max temp hp. I will update this PR accordingly to my understanding of what is needed. If anyone disagrees with me here we can happily discuss or maybe just leave it as is.